### PR TITLE
fix pdf build

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**

### CODE REVIEW

**Comment by Samuelopez-ansys** [comment]
Hi @Revathyvenugopal162 ,

Is this PR completed? We would like to launch a new release this week, and it would be great if the PDF creation would work. 

Thank you!

**Comment by Revathyvenugopal162** [comment]
@Samuelopez-ansys  The pdf build is failing at some point which is not very usual, i will investigate about this and update this PR as soon as possible. https://github.com/ansys/pyaedt/actions/runs/6302378169/job/17109436879#step:10:6814

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM

**Review by MaxJPRey — APPROVED** [review]
This is great @Revathyvenugopal162

**Review by maxcapodi78 — APPROVED** [review]

File: doc/source/conf.py [hunk 1]
```diff
@@ -11,23 +11,31 @@
...
+                                watermark, 
+                                ansys_logo_white, 
+                                ansys_logo_white_cropped, latex)
 from importlib import import_module
 from pprint import pformat
 from docutils.parsers.rst import Directive
 from docutils import nodes
 from sphinx import addnodes
+
+# <-----------------Override the sphinx pdf builder---------------->
```
[the remark this answers is not in the document]
Comment (Revathyvenugopal162) [reply doc/source/conf.py:25]: Sure i will add that.

**Comment by Revathyvenugopal162** [comment]
merging this! Thank you all for the review

**Pull request merged**